### PR TITLE
SDCORE-747: Adding repository path to module name

### PR DESCRIPTION
**Pull request by gajjarvini**
1) Changed module name from "gnbsim" to "github.com/omec-project/gnbsim"
    2) Removed independent profile files

### CODE REVIEW

File: factory/config.go [hunk 1]
```diff
@@ -11,14 +11,15 @@ package factory
...
+
+	"github.com/omec-project/gnbsim/gnodeb/context"
+	gnbctx "github.com/omec-project/gnbsim/gnodeb/context"
+	profctx "github.com/omec-project/gnbsim/profile/context"
 )
 
 const (
 	GNBSIM_EXPECTED_CONFIG_VERSION string = "1.0.0"
-	GNBSIM_DEFAULT_CONFIG_PATH            = "/gnbsim/config/gnb.conf"
+	GNBSIM_DEFAULT_CONFIG_PATH            = "/github.com/omec-project/gnbsim/config/gnb.conf"
```
[the remark this answers is not in the document]
Comment (gajjarvini) [reply factory/config.go:22]: You are correct, This got updated in "find and replace
"

File: factory/config.go [hunk 2]
```diff
@@ -11,9 +11,10 @@ package factory
 
 import (
 	"fmt"
-	"gnbsim/gnodeb/context"
-	gnbctx "gnbsim/gnodeb/context"
-	profctx "gnbsim/profile/context"
+
+	"github.com/omec-project/gnbsim/gnodeb/context"
```
Comment (thakurajayL) [inline factory/config.go:15]: Why we need line #15 & also #16 ?

[the remark this answers is not in the document]
Comment (gajjarvini) [reply factory/config.go]: Yes we have now removed the reference for "context", instead we are using only "gnbctx"

**Comment by gajjarvini** [comment]
> Could you please respond to my comment.

Hi Ajay, I have now made requested changes

**Review by vthiruveedula — COMMENTED** [review]
+1 to these changes.

**Pull request merged**